### DISCUSSION TITLE
Creating a PVModule

### DISCUSSION THREAD

**Discussion by AugustVan** [SunPower/PVMismatch#157] [Q&A]
Hello,

I am currently working on a project were I have to simulate the power curve and IV-curve of a solar panel under the influence of different shadows. To do this I created a simulation (with bifacial radiance) where I can simulate the irradiance on every cell of the solar panel I use (132 cells). Now I want to use these values to recreate the power curve and IV-curve and I think I can achieve this with PVMismatch. Now the problem is that I can't seem to succeed in making the PVmodule I need. The module I use can be found below, this is a screenshot of LTSpice where I have already created the module. The bypass diodes are surrounded in red because they are hard to see on the picture (they face left). The module exists out of two circuits that are connected in parallell. Both circuits exists out of 66 cells that are connected in series. There are also three bypass diodes that are situated betweend these two circuits. Is this possible to create in PVMismatch and if yes, how can I create this? Because I find it very difficult to work this out.

Kind Regards

August
![cell positions](https://github.com/SunPower/PVMismatch/assets/152192430/4c3b3e2a-aa6a-4ff5-93fd-d8633477fa9a)

**Comment by mikofski** [comment]
I think [this comment](https://github.com/SunPower/PVMismatch/issues/118#issuecomment-637211129) in #118  has an answer to a similar module with 144 hall cells: 
https://github.com/SunPower/PVMismatch/issues/118#issuecomment-637211129

**Reply by AugustVan** [reply]
Thx, I will have a look at it!
